components/WeeklyOverview/index.js

```diff
@@ -212,6 +250,14 @@ const SupplementName = styled.div`
 
 const SupplementDosage = styled.div``;
 
+const ButtonContainer = styled.div`
+  display: flex;
+`;
+
+const CheckButton = styled.button`
```

Comment:
You declared some of your styled components twice. This will lead to errors when trying to compile your code.
You should remove the duplicates.

```suggestion
```